frontend/src/components/Navbar/Navbar.jsx

```diff
@@ -8,33 +9,28 @@ export default function Navbar({ user }) {
                 withCredentials: true
             });
             localStorage.clear();
-        } catch (error) {
-            console.log(error);
-        }
-    }
-
-    function handleLogin() {
-        window.location.href = "/login";
+        } catch (error) {}
     }
 
     return (
         <nav className="navbar">
             <div className="nav-pages">
-                <a href="/home">Stock Zone</a>
-                <a href="/home">Home</a>
-                <a href="/search">Search</a>
+                <Link to="/home">Stock Zone</Link>
+                <Link to="/home">Home</Link>
+                <Link to="/search">Search</Link>
             </div>
             <div className="nav-profile">
-                {!user && (
-                    <button onClick={handleLogin} className="log-in">
+                {user ? (
```

Comment:
If this is the same as `user !== null && user !== undefined`, then let's use that. Or the shorter form: `user != null` . 
It is a lot more explicit, so it's easier to understand what the check is for